packages/arb-ts/src/lib/l1Bridge.ts

```diff
@@ -213,4 +250,9 @@ export class L1Bridge {
     this.inboxCached = InboxFactory.connect(inboxAddress, this.l1Signer)
     return this.inboxCached
   }
+
+  public async getAndUpdateL1EthBalance(): Promise<BigNumber> {
+    const bal = await this.l1Signer.getBalance()
+    return (this.l1EthBalance = bal)
```

Comment:
this doesn't seem very _idiomatic_? The L-value gets returned after being assigned?